tests/test_dimension.py

```diff
@@ -170,6 +170,25 @@ def test_basic(self):
         assert np.all([np.allclose(usave.data[i], i*factor)
                       for i in range((nt+factor-1)//factor)])
 
+    def test_spacial_subsampling(self):
```

Comment:
docstring

Reply:
perhaps showing what code we aim to generate would be useful, because I can't decipher it from the code below (or at least, I'm not totally sure I'm getting it...)

Reply:
it's saving `u` every two grid points in both direction.